src/websocket.js

```diff
@@ -393,6 +393,12 @@ Strophe.Websocket.prototype = {
      * (Object) error - The websocket error.
      */
     _onError: function(error) {
+        // Workaround
+        // https://code.google.com/p/chromium/issues/detail?id=372776
+        if (this.socket.readyState === 0) {
+          return;
+        }
+
```

Comment:
Not sure I get the point.

From the bug report I gather that Chromium continues to send data frames to Strophe.js even after a close frame has been sent.

So what happens in strophe.js then, and what is the point of the above check?